### PR TITLE
Update website content for fully virtual event

### PR DESCRIPTION
**Pull request by etcadinfinitum**
All text content has been reviewed for phrasing and clarity re online-only transition. Added Zoom invites, Discord invite, and DevPost link.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,14 +53,18 @@
             </div>
 
             <div class="title">UWB Hacks the Cloud</div>
+            <div class="event">
+                <h4>April 17-19, 2020</h4>
```
Comment (Chris-Johnston) [inline index.html]: might still be good to include the kick-off time here, ending doesn't matter as much imo

File: index.html [hunk 2]
```diff
@@ -80,19 +87,19 @@ <h1>About UWB Hacks</h1>
                     students will come up with.</p>
             </div>
             <div><h2>Technical Tidbits</h2>
-                <p>On the day of the event, all of the following resources will be available to all attendees:</p>
+                <p>On the first day of the event (Friday April 17th), all of the following resources will be available to all attendees:</p>
```
Comment (Chris-Johnston) [inline index.html]: starting time

File: index.html [hunk 3]
```diff
@@ -151,22 +162,28 @@ <h2>Food, Friends, and Fun</h2>
...
+                    <tr><td>10:45</td><td>Talk: Introduction To Cloud Services (join via <a href="https://washington.zoom.us/j/951323956">Zoom</a>)</td></tr>
+                    <tr><td>13:15</td><td>Lightning Talks: Tales from the Cloud (join via <a href="https://washington.zoom.us/j/573514052">Zoom</a>)</td></tr>
                     <tr><td>14:45</td><td>Workshop: TBA</td></tr>
-                    <tr><td>18:00</td><td>Free Dinner</td></tr>
-                    <tr><td>20:00</td><td>Submissions Due on DevPost</td></tr>
-                    <tr><td>20:15</td><td>Ice Cream Sundae Bar</td></tr>
-                    <tr><td>20:45</td><td>Demo Presentations</td></tr>
-                    <tr><td>21:45</td><td>Awards Ceremony</td></tr>
+                    <tr><td>20:00</td><td>Mentors go offline</td></tr>
+                </table>
```
Comment (Chris-Johnston) [inline index.html]: Maybe include an entry for Saturday which is just "Hacking time"

File: index.html [hunk 4]
```diff
@@ -178,12 +195,24 @@ <h1>Event Schedule</h1>
...
-                <p>UWB Hacks the Cloud will be happening on April 17, 2020 regardless!</p>
-                <p><strong>All workshops, technical talks, and presentations will be facilitated through Zoom.</strong> The Zoom invites will be available to all registered attendees in early April.</p>
-                <p><strong>All registered participants are welcome to attend the hackathon virtually.</strong> In the event that we can host the event on-campus, we will arrange for hybrid attendance so people can participate remotely.</p>
-                <p>The organizers are currently working with UW Bothell administrators to arrange for an in-person hackathon for those who wish to physically attend, assuming that the school okays it. <strong>We hope that this will be the case, and we will be updating all participants and volunteers about how campus closures or restrictions might affect the event.</strong></p>
-                <p>We will work hard to make sure you can participate regardless of whether you are self-quarantining or otherwise concerned about getting sick. Our collective health is our most valuable possession.</p>
+                <h2>What about COVID-19?</h2>
```
Comment (Chris-Johnston) [inline index.html]: might want to add a tidbit about how swag/goodies will work given that this is now all virtual

File: schedule.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+function strikethroughschedule() { //cross out schedule events as day progresses 
+    var today = new Date();
+    // TESTING ONLY:
+    // var today = new Date(2020, 3, 21, 18, 1);
+    var date = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+today.getDate();
+    
+    if(date < "2020-4-17"){return;} //check for hackathon date, exit if not hackathon
+    
+    var currenttime = today.getHours() + ":" + today.getMinutes()
+    console.log('Date & time: ' + date + ' ' + currenttime);
```
Comment (Chris-Johnston) [inline schedule.js]: probably can remove this line

File: index.html [hunk 6]
```diff
@@ -151,22 +162,28 @@ <h2>Food, Friends, and Fun</h2>
...
                     <!-- Use HTML entity for table column spacing -->
-                    <tr><td>10:00 &nbsp; &nbsp;</td><td>Team Formation Mixer, hosted by ACM-W</td></tr>
+                    <tr><td>10:00</td><td>Team Formation Mixer, hosted by ACM-W</td></tr>
                     <tr><td>10:00</td><td>Hacking Begins</td></tr>
-                    <tr><td>10:45</td><td>Talk: Introduction To Cloud Services</td></tr>
-                    <tr><td>12:30</td><td>Free Lunch</td></tr>
-                    <tr><td>13:15</td><td>Lightning Talks: Tales from the Cloud</td></tr>
+                    <tr><td>10:00</td><td>Mentors go online (join via <a href="https://discord.gg/EDBbxmt">Discord</a>)</td</tr>
+                    <tr><td>10:45</td><td>Talk: Introduction To Cloud Services (join via <a href="https://washington.zoom.us/j/951323956">Zoom</a>)</td></tr>
+                    <tr><td>13:15</td><td>Lightning Talks: Tales from the Cloud (join via <a href="https://washington.zoom.us/j/573514052">Zoom</a>)</td></tr>
```
Comment (Chris-Johnston) [inline index.html]: not too familiar with zoom, are these supposed to be different links?

Comment (etcadinfinitum) [reply index.html]: Yes, I set them up as different meetings. I suppose they _could_ all be the same meeting. Will think about that.

File: index.html (hunk 1 above)
Comment (cforcomputer) [reply index.html]: As long as the schedule is easily accessible I don't think it needs the start time here.

**Comment by Chris-Johnston** [comment]
> Push first workshop time to 11AM instead of 10:45
huzzah, given that there's more time I don't mind things starting a little bit later

**Pull request merged**